docs/graphql/manual/guides/deployment/render.rst

```diff
@@ -0,0 +1,115 @@
+Hasura GraphQL Engine on Render
+====================================================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+This guide talks about how to deploy the Hasura GraphQL engine on `Render
+<https://render.com>`__.
+
+
+Pre-requisites
+--------------
+
+- GitHub (or alternative) account
```

Comment:
What is meant by "alternative"?

Reply:
Gitlab, mostly, but possibly Bitbucket as well. I didn't take the time to confirm it is the same process. We could leave the bit about alternates out.

Reply:
`GitHub` or `GitLab` account.